hr_timesheet_sheet/static/src/scss/backend.scss

```diff
@@ -1,9 +1,46 @@
-.hr_timesheet_sheet_summary .o_field_x2many_2d_matrix table {
-    tbody td {
-        text-align: left;
+.hr_timesheet_sheet_summary .o_field_x2many_2d_matrix {
+    .table-responsive {
+        max-height: 450px;
```

Comment:
I'm no web-frontend developer, but how this will work out on mobiles (if that question even qualifies)

Reply:
Mobile view is broken: https://github.com/OCA/web/pull/1404

Reply:
Mobile view is broken: https://github.com/OCA/web/pull/1404